tensorboard/plugins/graph/tf_graph_common/util.ts

```diff
@@ -218,16 +224,21 @@ module tf.graph.util {
    * Returns the human readable version of the unit.
    * (e.g. 1.35 GB, 23 MB, 34 ms, 6.53 min etc).
    */
-  export function convertUnitsToHumanReadable(value, units, unitIndex) {
-    unitIndex = unitIndex == null ? 0 : unitIndex;
+  export function convertUnitsToHumanReadable(
+      value: number, units: Unit[], unitIndex: number = 0) {
```

Comment:
Should be `ReadonlyArray<Unit>`?

Reply:
Did not know about `ReadonlyArray`. This seems very useful for defining API in the future. I love it. Thanks!

Reply:
Sure thing. Yes, it’s very useful! Because `Array<T>` is a subclass of
`ReadonlyArray<T>`, most functions can freely accept `ReadonlyArray<T>`
to allow clients to pass either variety of array and still have the
guarantee that the function isn’t mutating its argument.